src/components/Video/index.js

```diff
@@ -80,10 +81,12 @@ class Video extends Component<Props, State> {
   handleRecordingStop = () => {
     const { switchSeconds, hasRecordingTakenTooLong } = this.state
     const { challenges, challengesId: id } = this.props
+    const webcam = this.webcam
```

Comment:
[minor] I find this sort of thing makes things harder to read. `this.webcam` makes it explicit that its bound to `this`. Seeing just `webcam` makes me think it's locally scoped

I always find it a needless way to save 5 characters (or avoid binding issues, but it doesn't look like that's what you're doing)

Reply:
For some reasons, `flow` started complaining about `this.webcam.getVideoBlob()` which was already there before my change but was not returning an error. Assigning `this.webcam` to a variable stops `flow` throwing an error. I don't know what the best way to fix this is...

Reply:
What error does it throw? I'm sure there's a better way to resolve flow's issue